app/assets/stylesheets/pages/_home.scss

```diff
@@ -1 +1,7 @@
 // Specific CSS for your home-page
+.devise-wrapper {
```

Comment:
you need to create a different file for your css, this is the one for the homepage, you can create a file called "_devise.scss" inside the folder "pages", then in the file "_index.scss" in the same folder, add a lign : @import "devise";

Reply:
@ShadyaJ I fixed this. Please review. Thank you.